assets/js/admin/orders/order-overview/models/state.js

```diff
@@ -71,51 +71,19 @@ export const State = Backbone.Model.extend(
 		 * @return {number} Order subtotal.
 		 */
 		getSubtotal() {
-			const { models: items } = this.get( 'items' );
-
 			// Use stored value if the record has already been created.
 			if ( false === this.get( 'isAdding' ) ) {
-				// Use the `discount` column for each OrderItem. OrderAdjustments do
-				// not store the per-item discount amount.
-				//
-				// Used when viewing a Refund to find the actual amount refunded per item.
-				//
-				// A $50.00 OrderItem that has a $12.50 discount applied by splitting two
-				// flat rate discounts of $5 and $10 would incorrectly try to apply $15.00
-				// discount if the attached OrderAdjustments were summed.
-				return items.reduce(
-					( amount, item ) => {
-						return amount += ( item.get( 'subtotal' ) - item.get( 'discount' ) );
-					},
-					0
-				);
+				return this.get( 'order' ).subtotal;
 			}
 
-			// Add all subtotals.
-			const subtotal = items.reduce(
+			const { models: items } = this.get( 'items' );
+
+			return items.reduce(
 				( amount, item ) => {
 					return amount += +item.get( 'subtotal' );
```

Comment:
Can you explain the `+item.get( 'subtotal' )` use here?  I've used reduce before in vanilla JS, but I've never seen this syntax.

Reply:
@LisaCee That ensures the value is a number when adding it to the previous value. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Unary_plus